google-cloud-storage/src/main/java/org/jclouds/googlecloudstorage/domain/templates/ObjectTemplate.java

```diff
@@ -37,8 +37,8 @@
    private String contentType;
    private String crc32c;
    private String md5Hash;
-   private Map<String, String> metadata = Maps.newLinkedHashMap();
-   private List<ObjectAccessControls> acl = Lists.newArrayList();
+   private Map<String, String> metadata;
+   private List<ObjectAccessControls> acl;
```

Comment:
Initializing these as empty overwrites metadata and acls when using the PATCH operation.